arkouda-env-dev.yml

```diff
@@ -1,20 +1,20 @@
 name: arkouda-env-dev
 channels:
   - conda-forge
+  - defaults

```

Comment:
i had more luck when testing this locally with the order of the channels swapped

Reply:
Mine works without issue with them in this order. The order should not matter